app/src/main/java/edu/uc/app/parkfinder/ui/main/MainFragment.kt

```diff
@@ -25,4 +23,7 @@ class MainFragment : Fragment(R.layout.main_fragment) {
         })
 
     }
+    companion object {
+        fun newInstance() = MainFragment()
+    }
```

Comment:
Companion object moved to the end

Reply:
This duplicates a previous code review. When that occurs, first in wins.